data/libs/encounters_lib.lua

```diff
@@ -313,6 +313,20 @@ function Encounter:addBroadcast(message, type)
 	})
 end
 
+---Adds a stage that message sends message to players and webhook for server events
+---@param title string The title of the message
+---@param message string The message to send
+---@return boolean True if the message stage is added successfully, false otherwise
+function Encounter:addServerBroadcast(title, message, type)
```

Comment:
`Encounter:addBroadcast` already exists, and if the `Encounter` is `global` it will broadcast to all players.

Please change `Encounter:broadcast` to send the Webohook` instead of adding a new method, having two `addBroadcasts` is not necessary I don't think.

Reply:
I added the function since the I also wanted a title